packages/experimental/Tabs/src/Tabs.types.ts

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { View, ViewProps } from 'react-native';
+import { TextProps } from '@fluentui-react-native/experimental-text';
+import { FocusZoneProps } from '@fluentui-react-native/focus-zone';
+import { FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IShadowTokens } from '@fluentui-react-native/tokens';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
+
+export const tabsName = 'Tabs';
+
+export interface TabsContextData {
+  /*
+   ** The currently selected TabsItem's key
+   */
```

Comment:
nit: update all comment formats with

```
/**
 * The currently selected TabsItem's key
 */
```
  
instead of

```
/*
 ** The currently selected TabsItem's key
 */
```